code/game/data_huds.dm

```diff
@@ -133,7 +133,7 @@ proc/process_sec_hud(var/mob/M, var/advanced_mode,var/mob/eye)
 					if("*Arrest*")
 						holder.icon_state = "hudwanted"
 					if("Incarcerated")
-						holder.icon_state = "hudprisoner"
+						holder.icon_state = "hudincarcerated"
```

Comment:
i hope you changed the icon name in the dmi as well

Reply:
I have, I just forgot to put it in the commit. The prisoner JOB icon is technically unused but I gave the job title one that's similar to the warden bars but black and white. The incarcerated icon has been renamed to fit the code change as well since the names would conflict.